### DISCUSSION TITLE
provide extra arguments for custon aggergation function

### DISCUSSION THREAD

**Discussion by sgratzl** [TanStack/table#2645] [General]
according to the docs the aggregation function has the following signature: 
```js
aggregate: String | Function(leafValues, aggregatedValues) => any
```
https://react-table-omega.vercel.app/docs/api/useGroupBy

I'm currently implementing some custom aggregation functions and I would needs access to the column itself. e.g. the `aggregateValue` function gets as an additional argument the column. I'd love to have such an argument also in this function. My use case is that, I'm also computing some global statistics for each column and want to access them while creating the aggregate.

**Comment by ViieeS** [comment]
In my case I use external  parents object for aggregated values, so I need groupByID, groupByVal, column. The only way I found is to use Aggregated renderer, but it's quite tricky.
